sdcm/cluster_docker.py

```diff
@@ -81,11 +83,19 @@ def __init__(self,
                          base_logdir=base_logdir,
                          node_prefix=node_prefix)
         self.node_index = node_index
+        self.docker_service_manager = None
+        self._docker_cmd_runner = None
 
         if container is not None:
             assert int(container.labels["NodeIndex"]) == node_index, "Container labeled with wrong index."
             self._containers["node"] = container
 
+    def setup_docker_management(self, container: Container):
+        """Set up Docker command runner and service manager for direct container management"""
+        self._docker_cmd_runner = RemoteDocker(
```

Comment:
```suggestion
        self.remoter = RemoteDocker(
```

and put that in `_init_remoter`  so there won't be any ssh, and then it would surface those places needs attention

Reply:
also I don't see where you remove the part it's building the docker, we should ditch the whole thing even throwing out the whole `NodeContainerMixin`

Reply:
Removed building the scylla-sct docker image